analyzer/libretime_analyzer/status_reporter.py

```diff
@@ -94,55 +86,48 @@ def process_http_requests(ipc_queue, http_retry_queue_path):
-        )
+def send_http_request(picklable_request: PicklableHttpRequest, retry_queue):
     try:
         bare_request = picklable_request.create_request()
-        s = requests.Session()
-        prepared_request = s.prepare_request(bare_request)
-        r = s.send(prepared_request, timeout=StatusReporter._HTTP_REQUEST_TIMEOUT)
-        r.raise_for_status()  # Raise an exception if there was an http error code returned
+        session = requests.Session()
+        prepared_request = session.prepare_request(bare_request)
+        resp = session.send(
+            prepared_request, timeout=StatusReporter._HTTP_REQUEST_TIMEOUT
+        )
+        resp.raise_for_status()  # Raise an exception if there was an http error code returned
         logger.info("HTTP request sent successfully.")
-    except requests.exceptions.HTTPError as e:
-        if e.response.status_code == 422:
+    except requests.exceptions.HTTPError as exception:
+        if exception.response.status_code == 422:
             # Do no retry the request if there was a metadata validation error
-            logger.error(
-                "HTTP request failed due to an HTTP exception. Exception was: %s"
-                % str(e)
-            )
+            logger.error(f"HTTP request failed due to an HTTP exception: {exception}")
         else:
             # The request failed with an error 500 probably, so let's check if Airtime and/or
             # the web server are broken. If not, then our request was probably causing an
             # error 500 in the media API (ie. a bug), so there's no point in retrying it.
-            logger.error("HTTP request failed. Exception was: %s" % str(e))
-            parsed_url = urlparse(e.response.request.url)
+            logger.error(f"HTTP request failed: {exception}")
+            parsed_url = urlparse(exception.response.request.url)
             if is_web_server_broken(parsed_url.scheme + "://" + parsed_url.netloc):
                 # If the web server is having problems, retry the request later:
                 retry_queue.append(picklable_request)
                 # Otherwise, if the request was bad, the request is never retried.
                 # You will have to find these bad requests in logs or you'll be
                 # notified by sentry.
-    except requests.exceptions.ConnectionError as e:
+    except requests.exceptions.ConnectionError as exception:
         logger.error(
-            "HTTP request failed due to a connection error. Retrying later. %s" % str(e)
+            f"HTTP request failed due to a connection error. Retrying later. {exception}"
         )
         retry_queue.append(picklable_request)  # Retry it later
-    except Exception as e:
-        logger.error("HTTP request failed with unhandled exception. %s" % str(e))
-        logger.error(traceback.format_exc())
+    except Exception as exception:
+        logger.error(f"HTTP request failed with unhandled exception. {exception}")
```

Comment:
You're double logging the exception here?

Reply:
Yup my bad, I did some dumb replacement without thinking much.